pages/patterns/delete.js

```diff
@@ -1,108 +1,126 @@
-import React from 'react';
+/* eslint-disable react/jsx-key */
+import React, { useState } from 'react';
 import { Heading } from '@twilio-paste/core/heading';
 import { Text } from '@twilio-paste/core/text';
 import { Paragraph } from '@twilio-paste/core/paragraph';
 import { useToaster, Toaster } from '@twilio-paste/core/toast';
+import { Alert } from '@twilio-paste/core/alert';
+import { Anchor } from '@twilio-paste/core/anchor';
+import { Box } from '@twilio-paste/core/box';
 
-import { DeleteTable } from '../../components/site/patterns/DeleteTable';
 import { DeleteConfirm } from '../../components/site/patterns/DeleteConfirm';
+import { DeleteTable } from '../../components/site/patterns/DeleteTable';
 
-export default function Delete() {
-  const severity = 'medium';
+/*
+ * To change severity level (for more information on that, check out Paste Delete Pattern docs),
+ * type 'low' or 'medium' below ('high' severity is not yet available).
+ */
 
-  const [data, setData] = React.useState([
-    {
-      id: 1,
-      friendlyName: (
-        <Text as="span" fontFamily="fontFamilyText">
-          Fiiiiiiiiirst thing
-        </Text>
-      ),
-      sid: (
-        <Text as="span" fontFamily="fontFamilyCode">
-          4392908903
-        </Text>
-      ),
-    },
-    {
-      id: 2,
-      friendlyName: (
-        <Text as="span" fontFamily="fontFamilyText">
-          Second thing
-        </Text>
-      ),
-      sid: (
-        <Text as="span" fontFamily="fontFamilyCode">
-          8439025738
-        </Text>
-      ),
-    },
-    {
-      id: 3,
-      friendlyName: (
-        <Text as="span" fontFamily="fontFamilyText">
-          Third thing
-        </Text>
-      ),
-      sid: (
-        <Text as="span" fontFamily="fontFamilyCode">
-          8439025738
-        </Text>
-      ),
-    },
-    {
-      id: 4,
-      friendlyName: (
-        <Text as="span" fontFamily="fontFamilyText">
-          last thing
-        </Text>
-      ),
-      sid: (
-        <Text as="span" fontFamily="fontFamilyCode">
-          6239485739
-        </Text>
-      ),
-    },
-  ]);
+// To change what appears on the Delete Pattern page, edit the data below.
+
+const severity = 'medium';
+const pageHeading = 'Services';
+const tableDescription =
+  'Services enable you to organize and identify your conversations by use case, or manage them in multiple environments (e.g. dev, stage, prod). This information held within a service is siloed, protecting both your recipients&apos; data.';
+const emptyState = 'Nothing to delete here!';
+const columns = ['friendlyName', 'sid', 'greetings'];
+const rows = [
```

Comment:
Love all this! I think having it in one place will make the designers more likely to edit the thing they want. Great job!!

Reply:
also, such well named variables <3